### PR TITLE
ENG-640 version details

### PR DESCRIPTION
**Pull request by ichalagashvili**
This PR implements:

- Content Version Details Page (rendering all attribute values)
- Missing Top Table for Single Content's versions history page
- Exporting common functions (such as: calculating icon color and title)

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/state/versioning/reducer.js [hunk 1]
```diff
@@ -35,8 +38,18 @@ export const versioningMap = (state = {}, action = {}) => {
   }
 };
 
+export const detailedContentVersion = (state = {}, action = {}) => {
```
Comment (gidesan) [inline src/state/versioning/reducer.js:41]: why not calling it "contentVersionDetails"?

File: src/ui/versioning/details/ContentVersionDetailsValueItem.js [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+  } else if (contentDest != null) {
+    icon = 'file-text-o';
+    linkToText = 'a content';
+    url = contentDest;
+  }
+  return (
+    <FormGroup>
+      {showLabel && <label>{code}</label>}
+      <div>
+        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
```
Comment (gidesan) [inline src/ui/versioning/details/ContentVersionDetailsValueItem.js:145]: can we refactor this using a more a11y-friendly approach?

File: src/ui/versioning/details/ContentVersionDetailsValueItem.js [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+        {renderTarget}
+      </div>
+    </FormGroup>
+  );
+};
+
+const renderMonolistAttributeValue = (attribute, defaultLangCode, currentLangCode, domain) => {
+  const { code, elements } = attribute;
+  const renderedCompositeValues = elements.map((attr, i) => (
+    // eslint-disable-next-line react/no-array-index-key
```
Comment (gidesan) [inline src/ui/versioning/details/ContentVersionDetailsValueItem.js:199]: please satisfy eslint :)

File: src/ui/versioning/details/ContentVersionDetailsValueItem.js [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+    <FormGroup key={`${code}-${attr.code}-id-${i}`} style={{ marginLeft: '10px' }}>
+      <span className="label label-default" style={{ marginBottom: '10px', display: 'inline-block' }}>
+        {i + 1}
+      </span>
+      <span>
+        { /* eslint-disable-next-line no-use-before-define */ }
```
Comment (gidesan) [inline src/ui/versioning/details/ContentVersionDetailsValueItem.js:205]: please satisfy eslint :)

File: src/ui/versioning/details/ContentVersionDetailsValueItem.js [hunk 5]
```diff
@@ -0,0 +1,293 @@
...
+      key={`${code}-id-${attr.code}`}
+      compoundExpand={false}
+      compoundExpanded={false}
+      description={(
+        <FormGroup>
+          <span className="label label-default" style={{ marginBottom: '10px', display: 'inline-block' }}>
+            {attr.code}
+          </span>
+          <div>
+            { /* eslint-disable-next-line no-use-before-define */ }
```
Comment (gidesan) [inline src/ui/versioning/details/ContentVersionDetailsValueItem.js:67]: please satisfy eslint :)

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-cms/pull/203#pullrequestreview-433638544" title="Approved">BrenoQVDS</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-cms).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-cms/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**